site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst

```diff
@@ -57,3 +52,60 @@ Install Tor
         brew services start tor
 
 This will start Tor and ensure that it is always running, even after a restart.  See the `Tor Project docs <https://2019.www.torproject.org/docs/tor-doc-osx.html.en>`_ for more details.
+
+Enable Tor Systemwide
+----------------------
+
+#. Now enable apache service:
+
+    .. code-block::
+
+        sudo launchctl load -w /System/Library/LaunchDaemons/org.apache.httpd.plist
+
+#. Enable proxy autoconfig file:
```

Comment:
This should probably say something like "Download the Start9 standard proxy config file, or use your own"

Reply:
Resolved